app/models/background.rb

```diff
@@ -44,11 +44,23 @@ def summarize
     results
   end
 
+  def pipeline_run_ids
+    updated_pipeline_run_ids = []
+    pipeline_runs.each do |pr|
+      pr_new = pr.sample.pipeline_runs.find_by(job_status: PipelineRun::STATUS_CHECKED)
```

Comment:
can you add a comment here about what's going on? i think it's important to explain:
- find_by is returning the first result
- the first result is always the most recent *completed* run because of the way the table is indexed, which we're assuming to be with the latest pipeline ID (though this may not be the case)
- we might change this function down the road if we allow backgrounds to be assembled using older pipeline runs of a sample